src/index.tsx

```diff
@@ -68,7 +70,7 @@ export const FullScreen: React.FC<FullScreenProps> = ({
 
   return (
     <div
-      className={classNames.join(' ')}
+      className={[className, ...classNames].join(' ')}
```

Comment:
@pomle  btw if you want to be perfect you need to filter nulls and undefined from the classnames
currently, if we don't pass classname, the class name will be " fullscreen" 

`[null, "fullscreen"].join(' ')` => ' fullscreen'
`[null, "fullscreen"].filter(classname => !!classname).join(' ')` => 'fullscreen'